LanguageFeatures/Private-named-parameters/static_semantics_A02_t01.dart

```diff
@@ -19,8 +19,8 @@
 class C {
   String _p;
   String p;
-  C({this._p = "", int? _p}) : p = "";
-//                      ^^
+  C(int? _p, {this._p = "", }) : p = "";
```

Comment:
The formatter will probably remove that comma:
```suggestion
  C(int? _p, {this._p = ""}) : p = "";
```